src/SourceBuild/tarball/content/src/source-mappings.json

```diff
@@ -0,0 +1,140 @@
+{
+    "defaults": {
+        "defaultRef": "main",
+        "exclude": [
+            "**/*.dll",
+            "**/*.Dll",
+            "**/*.exe",
+            "**/*.pdb",
+            "**/*.mdb",
+            "**/*.zip",
+            "**/*.nupkg"
+        ]
+    },
+    "mappings": [
+        {
+            "name": "arcade",
+            "defaultRemote": "https://github.com/dotnet/arcade",
+            "include": [
+                "*.*",
+                ".*",
+                "eng/**/*",
+                "src/**/*",
+                "scripts/**/*"
+            ],
+            "exclude": [
+                "**/tests/**/*",
+                "**/*.Tests/**/*"
+            ]
+        },
+        {
+            "name": "aspnetcore",
+            "defaultRemote": "https://github.com/dotnet/aspnetcore"
```

Comment:
Do you have a plan on how we'll handle internal builds (when the remote that we're updating from will be the internal repo). Will that not matter?

Reply:
Is this a scenario that I should get working right away?

Reply:
Nope, since we're targeting .NET 8 first, which wouldn't contain security fixes till rc1 or

Reply:
Can you open an issue for it?

Reply:
Done - https://github.com/dotnet/arcade/issues/10549